DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/AzureBlobStorageMultipartProvider .cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public class AzureBlobStorageMultipartProvider : MultipartFileStreamProvider
```

Comment:
Interesting case. But why you did't accept parameters into action body? Instear using Request property of ApiController? As I understand Web Api allows to have mixed parameters inside action signature.